bash_scripts/configuration.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * @package    Joomla.Installation
+ *
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ *
+ * -------------------------------------------------------------------------
+ * THIS SHOULD ONLY BE USED AS A LAST RESORT WHEN THE WEB INSTALLER FAILS
+ *
+ * If you are installing Joomla! manually ie not using the web browser installer
+ * then rename this file to configuration.php eg
+ *
+ * UNIX -> mv configuration.php-dist configuration.php
+ * Windows -> rename configuration.php-dist configuration.php
+ *
+ * Now edit this file and configure the parameters for your site and
+ * database.
+ *
+ * Finally move this file to the root folder of your Joomla installation eg
+ *
+ * UNIX -> mv configuration.php ../
+ * Windows -> copy configuration.php ../
+ *
+ */
+class JConfig
+{
+	/* Site Settings */
+	public $offline = '0';
+	public $offline_message = 'This site is down for maintenance.<br> Please check back again soon.';
+	public $display_offline_message = '1';
+	public $offline_image = '';
+	public $sitename = 'Joomla!';            // Name of Joomla site
+	public $editor = 'tinymce';
+	public $captcha = '0';
+	public $list_limit = '20';
+	public $access = '1';
+	public $frontediting = '1';
+
+	/* Database Settings */
+	public $dbtype = 'mysqli';               // Normally mysqli
+	public $host = '127.0.0.1';              // This is normally set to localhost
+	public $user = 'root';                   // DB username
+	public $password = '${DBPASSWORD}';      // DB password
+	public $db = '${DBNAME}';                // DB database name
+	public $dbprefix = 'jos_';               // Do not change unless you need to!
+
+	/* Server Settings */
+	public $secret = 'FBVtggIk5lAzEU9H';     // Change this to something more secure
+	public $gzip = '0';
+	public $error_reporting = 'default';
+	public $helpurl = 'https://help.joomla.org/proxy?keyref=Help{major}{minor}:{keyref}&lang={langcode}';
+	public $ftp_host = '';
+	public $ftp_port = '';
+	public $ftp_user = '';
+	public $ftp_pass = '';
+	public $ftp_root = '';
+	public $ftp_enable = '0';
+	public $tmp_path = '/shared/httpd/${INSTANCEFOLDER}/htdocs/administrator/logs';
```

Comment:
these paths are a bit specific in the `/shared/httpd` path, or is that something of devilbox?